### PR TITLE
Fixed lint warnings in RxJava example

### PR DESCRIPTION
**Pull request by cmelchior**
@realm/java

### CODE REVIEW

File: examples/rxJavaExample/src/main/res/values/list_default_description.xml [hunk 1]
```diff
@@ -0,0 +1,3 @@
+<?xml version="1.0" encoding="utf-8"?>
```
Comment (zaki50) [inline examples/rxJavaExample/src/main/res/values/list_default_description.xml]: no need to add this file

File: examples/rxJavaExample/lint.xml [hunk 2]
```diff
@@ -0,0 +1,7 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<lint>
+    <issue id="AllowBackup" severity="ignore" />
+    <issue id="GoogleAppIndexingWarning" severity="ignore" />
+    <issue id="IconMissingDensityFolder" severity="ignore" />
+    <issue id="IconLauncherShape" severity="ignore" />
+</lint>
```
Comment (nhachicha) [inline examples/rxJavaExample/lint.xml]: add new line please

File: examples/rxJavaExample/src/main/res/values/list_default_description.xml [hunk 3]
```diff
@@ -0,0 +1,3 @@
...
+<resources>
+</resources>
```
Comment (nhachicha) [inline examples/rxJavaExample/src/main/res/values/list_default_description.xml]: please add new line

File: examples/rxJavaExample/lint.xml (hunk 2 above)
Comment (cmelchior) [inline examples/rxJavaExample/lint.xml]: Done

**Comment by stk1m1** [comment]
👍

**Comment by dalinaum** [comment]
:+1:

**Comment by zaki50** [comment]
:+1:

**Pull request merged**